### PR TITLE
Making  Vet clinic database

### PR DESCRIPTION
**Pull request by mrkamin**
What does this PR do?
In this PR I create a Database ...

Description of the task to be completed 👎

In this project, we will use a relational database to create the initial data structure for a vet clinic. we will create a table to store animals' information, insert some data into it, and query it.
 

Feel free to check the query result screenshot [Screenshot Link] (https://github.com/mrkamin/Vet-Clinic/tree/create-animals-table/quiry-result).

### CODE REVIEW

File: quires.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (TuyishimireEric) [inline quires.sql:2]: - [x] **Good job so far 👏 👏** However, it was suggested in the project requirements to add a screenshot of all query results to the pull request description. 😂 Kindly, add them as the result of all your queries 👍
 
image

File: InsertValue.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Agumon', DATE '2020-02-03', 0, true, 10.23);
```
Comment (TuyishimireEric) [inline InsertValue.sql]: - [x] Kindly in this project we are required to copy all the SQL used in into the `data.sql` file in the generated repository, not `InsertValue` 😀😀

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @mrkamin  👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlight 
✔ correct SQL commands👍 
✔ Professional readme👍 

## Required Changes ♻️

- check the comments below
**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: quires.sql (hunk 1 above)
Comment (mrkamin) [reply quires.sql:2]: Hi @TuyishimireEric,
thanks for your good and valuable feedback. 
as you mention I add the link on the PR description,

File: InsertValue.sql (hunk 2 above)
Comment (mrkamin) [reply InsertValue.sql]: Hi @TuyishimireEric,
thanks for your good and valuable feedback.
as you mentioned I change the Insert Value to data. SQL.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @mrkamin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@ShadyShawkat,
thanks for your good and valuable feedback.

**Pull request merged**